docs/src/ScrollableContainer.doc.js

```diff
@@ -591,4 +600,19 @@ function ScrollableContainerExample() {
   </MainSection>,
 );
 
+card(
+  <MainSection
+    name="Related"
+    description={`
+[Modal](/Modal) / [Sheet](/Sheet)
```

Comment:
I wouldn't worry about what you mentioned in the video, needing to mention the experimental nature of this. I think we should write these docs as they'll be when that experiment ships; since we're not giving devs control over the experimental group, there's no need to call it out here.

Reply:
ok